spec/steps/backend_steps.rb

```diff
@@ -68,7 +68,7 @@
 step 'I can add a new resource type' do
   click_link 'New resource type'
 
-  fill_in 'Resource type name', with: 'Virtual PC'
+  fill_in 'Resource type name', with: 'Online PC'
```

Comment:
Why this?

Reply:
Because in the fixtures we had already declared a resource type with the name Virtual PC. In one of the new tests I've done, capibara throw an error because there were two values with Virtual PC.